taxonomies/ingredients.txt

```diff
@@ -9806,21 +9823,22 @@ es:queso de leche pasteurizada
 
 <en:cheese
 en:cheese powder
-ca:formatge en pols
-da:ostepulver, ostpulver
-de:Käsepulver
-el:τυρί σε σκόνη
-es:queso en polvo
-fi:juustojauhe
-fr:fromage en poudre, poudre de fromage
-hu:sajtpor, porított sajt
-it:formaggio in polvere
-nl:kaaspoeder
-pl:ser w proszku
-pt:queijo em pó
-ru:Сырный порошок
-sk:sirotka v prahu
-sv:ostpulver
+#en:processing:en:powder
```

Comment:
I'm not sure I understand. Why do we keep en:cheese powder in the taxonomy? Is that because some languages don't have support for "powder" parsing yet? In that case we should leave those languages until we do have support I think.

Reply:
There are more examples like this. It is mostly due to children, which I can not delete. For instance when there is an associated CIQUAL.